sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3271,32 +3333,66 @@ pi_result piMemBufferCreate(pi_context Context, pi_mem_flags Flags, size_t Size,
   else
     Alignment = 1UL;
 
-  pi_result Result = PI_SUCCESS;
+  // If USM Import feature is enabled and hostptr is supplied,
+  // import the hostptr if not already imported into USM.
```

Comment:
I think this is a good place to explain in comment why we are doing so, and what's the cons/pros. Maybe also in the option doc.

Reply:
Please add more detail about how importing saves performance.